src/components/MovieCard.js

```diff
@@ -17,6 +19,9 @@ class MovieCard extends React.Component {
             </div>
             <div className="card-action">
```

Comment:
Por não conter nada nessa div, poderia já fechá-la na mesma linha. Algo assim:
```suggestion
            <div className="card-action" />
```